dist/test_data.py

```diff
@@ -260,4 +260,5 @@ def __ge__(self, other):
     'search_near_02' : Method(test_config),
     'search_near_03' : Method(test_config),
     'test_template' : Method(test_config),
+    'reverse_split' : Method(test_config),
```

Comment:
Looks like a missing A->Z 😉

Reply:
[ef0a708](https://github.com/wiredtiger/wiredtiger/pull/8554/commits/ef0a7084082cda3917b66d9453de9276b0151389) I also added a comment explaining what that section is and that it should be alphabetically ordered.